ophyd/tests/test_signal.py

```diff
@@ -592,3 +592,21 @@ def test_posix_paths(paths, path_signal):
 def test_path_semantics_exception():
     with pytest.raises(ValueError):
         EpicsPathSignal('TEST', path_semantics='not_a_thing')
+
+        
+@pytest.fixture(scope="module")
+def imported_ro_signal_class_from_pkg():
+    from ophyd import SignalRO
+    return SignalRO
+
+
+@pytest.fixture(scope="module")
+def imported_ro_signal_class_from_module():
+    from ophyd.signal import SignalRO
+    return SignalRO
+
+
+def test_import_ro_signal_class(
+        imported_ro_signal_class_from_pkg, 
+        imported_ro_signal_class_from_module):
+    assert imported_ro_signal_class_from_pkg == imported_ro_signal_class_from_module
```

Comment:
Maybe it can be done simpler without extra fixtures, such as:
```py
In [5]: from ophyd import SignalRO as SignalROFromPkg

In [6]: from ophyd.signal import SignalRO as SignalROFromModule

In [7]: SignalRO is SignalROFromModule
Out[7]: True
```

In my example I used the identity check (with `is`). I think your test can use that too.

Reply:
Sure, that works for me. Thanks for the guidance.  I'll make these changes.

For the glob import test, the following statement needs to be done at module level: `from ophyd import *`.  Is it ok for me to add a new file to the 'tests' folder for this purpose?  Or is there a better way to do that?

I don't want to pollute the namespace of the test_module.

Reply:
Actually, I'll try this with `from ophyd.__init__ import SignalRO`, so won't need a separate file.

`ophyd.__all__` is not defined; otherwise that should work also.

Reply:
Thanks! Regarding the star import, it's generally discouraged, and, as you rightly mentioned, it can pollute the namespace and can lead to confusion. I think the explicit imports would be sufficient for tests.